libs/api/domains/documents/src/lib/document.service.ts

```diff
@@ -223,4 +224,32 @@ export class DocumentService {
       }
     }
   }
+
+  async bulkMailAction(body: PostBulkMailActionInput): Promise<ActionMailBody> {
+    const messageIds = body.messageIds ?? []
+    const stringIds = messageIds.toString()
+    try {
+      await this.documentClient.bulkMailAction(
+        {
+          ids: messageIds,
+          action: body.action,
+          status: body.status,
+        },
+        body.nationalId,
+      )
+      return {
+        success: true,
+        messageId: stringIds,
+      }
+    } catch (e) {
+      logger.error('Post batch action failed', {
+        category: LOG_CATEGORY,
+        error: e,
+      })
```

Comment:
```suggestion
      logger.error('Post batch action failed', {
        category: LOG_CATEGORY,
        ...e,
      })
```
Do we want to log the action or the status to improve the debugging experience?